core/connector.ts

```diff
@@ -0,0 +1,7 @@
+import ConnectionToReplay from './connections/ConnectionToReplay';
```

Comment:
Should we fix this?

Reply:
Yes, but fixing this breaks Server.

I can remove it... and then remove related code in Server... but I thought it best to discuss flow first.

Reply:
Removed